addons/stock/__manifest__.py

```diff
@@ -82,6 +84,8 @@
         'views/stock_pack_operation_views.xml',
         'views/product_views.xml',
         'views/stock_config_settings_views.xml',
+        'views/report_stock_traceability.xml',
+        'views/stock_template.xml',
```

Comment:
this is commit from 46fc922448bd8ab1602055f2a754ea8e610b5866 and we don't have time to clean it with this pr, please ignore diff from this commit :)